upstream/solr6/local/92-init-solr.sh

```diff
@@ -192,6 +195,11 @@ function makeConfigs {
             setGlobalOptions "$CONFIG_FILE_CORE" archive
             escapeFile "$CONFIG_FILE_CORE"
 
+            # SOLR_BACKUP_DIR only useful for ASS v2.0.2* and up
+            if [ $SOLR_VERSION_MAJOR = 2 -a $SOLR_VERSION_REV != 0 -a $SOLR_VERSION_REV != 1 ]
```

Comment:
@Ranecdev You forgot to remove the version check here.

Reply:
Done